cluster-agent/controllers/util.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	appv1 "github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1"
 	"github.com/go-logr/logr"
-	"github.com/redhat-appstudio/managed-gitops/backend-shared/config/db"
```

Comment:
You can pull latest code from upstream since John's PR is merged with this fix.

Reply:
Yes , I will pull the latest changes, thanks